src/Sandbox.php

```diff
@@ -236,6 +240,8 @@ public function toArray(): array
             $result['databricks']['numberOfNodes'] = $this->databricksNumberOfNodes;
         }
 
+        $result['persistentStorage']['pvcName'] = $this->persistentStoragePvcName;
```

Comment:
so, unlike the others, it will return a null value instead of excluding it from the response?

Reply:
This is a bit tricky :) originally I had it the same way, setting `pvcName` only if `$this->persistentStoragePvcName` is set. But then I was not able to clear the flag on API. If you send `{ persistentStorage: {} }`, the API translates it as "`pvcName` was not changed" instead of "`pvcName` was removed"

Reply:
Yeah, I think it's ok.

but just because this is mainly @JakubMatejka 's baby :-) I'm curious what he thinks

Reply:
I don't think anything. I was fighting hard with the php and was happy that it worked at all.😄